manifests/guestcluster/1.18/pvcsi.yaml

```diff
@@ -147,6 +147,7 @@ spec:
             - "--supervisor-fss-namespace=$(CSI_NAMESPACE)"
             - "--fss-name=internal-feature-states.csi.vsphere.vmware.com"
             - "--fss-namespace=$(CSI_NAMESPACE)"
+            - "--use-gocsi=false"
```

Comment:
you are removing this arg and we default to `use-gocsi` `true`

https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/cmd/vsphere-csi/main.go#L43
> useGocsi             = flag.Bool("use-gocsi", true, "Flag to specify to use gocsi or not")

PR title says you want to disable gocsi in SV and PVCSI. How will that happen by removing this arg?

Reply:
I'm not removing the arg, I'm defaulting it to `false` from yamls. So it will set the `useGocsi` value in controller to `false`. It's like enabling the FSS.

Reply:
Thanks for the clarification. I miss read the diff.